.github/workflows/check_upcoming_browser_versions.yml

```diff
@@ -0,0 +1,50 @@
+# Reference on this file: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+# You need to add BrowserStack access keys in order for the tests to work. Get a BrowserStack automate account, then go
+# to the repository settings, the Secrets page and add secrets: BROWSERSTACK_USERNAME, BROWSERSTACK_ACCESS_KEY.
+# In order to receive notifications in Slack, create a Slack bot,
+# add the credentials to repository secrets SLACK_BOT_TOKEN (when installing the app) and SLACK_CHANNEL_ID (from within Slack).
+
+name: Check upcoming browser versions
+
+on: 
+  schedule:
+    - cron: '30 17 ? * MON-FRI'
```

Comment:
Let's use `0 19 ? * MON-FRI` to have a bigger gap between the scheduled pipelines

Reply:
Updated.